policyengine_us/parameters/gov/hhs/chip/children/max_age.yaml

```
@@ -0,0 +1,10 @@
+description: Maximum age for CHIP eligibility.
+metadata:
+  period: year
+  unit: /1
+  label: Maximum age for CHIP eligibility
+  reference:
+    - title: 4ec. 2110. [42 U.S.C. 1397jj] (a) Child Health Assistance - c(1)
+      href: https://www.ssa.gov/OP_Home/ssact/title21/2110.htm#:~:text=%284%29%20Low,family%20of%20the%20size%20involved
+values:
+  1966-01-01: 18
```

then use `<` to align with usc
```suggestion
  1966-01-01: 19
```